frontend/src/pages/cellguide/[cellTypeId].tsx

```diff
@@ -24,12 +16,12 @@ export const getServerSideProps: GetServerSideProps<{
   const { params } = context;
   const { cellTypeId: rawCellTypeId } = params ?? {};
   const cellTypeId = (rawCellTypeId as string)?.replace("_", ":");
+  const cellType = await fetchGptSeoDescription(cellTypeId);
```

Comment:
@tihuan This is a naked fetch and I don't think it is cached. Can we make our own cache by using a global object at the top level to save the fetchGptSeoDescription results per latest snapshot identifier and cell type? If the latest snapshot identifier changes, clear the cache. If the cell type exists in the top-level object, use its corresponding value instead of fetching.

Not sure how we would get the latest snapshot ID without needing to fetch, though.. I think something like this could work but we need to work out how to clear the cache when the snapshot changes.

Snapshot changes once a month, though, so we could just rely on new deployments to clear the cache.